lib/src/network/models/list_response.dart

```diff
@@ -24,5 +27,5 @@ class ListResponse<T extends Model> extends Equatable {
   Map<String, dynamic> toJson() => _$ListResponseToJson(this);
 
   @override
-  List<Object> get props => [data, meta ?? ''];
+  List<Object> get props => [data, meta ?? '', links ?? ''];
```

Comment:
`List<Object?> get props => [data, meta, links];`

Reply:
no yozbot, because this field is nullable

Reply:
oh sorry, I am not see this operator "?" after object